beautifultable/beautifultable.py

```diff
@@ -30,6 +30,9 @@
 import copy
 import csv
 import warnings
+from numpy import r_
+
+import pandas as pd
```

Comment:
Can we make `pandas` an optional dependency? so ideally just raise an exception inside the `to_df`, `from_df` methods if pandas is not installed. do make them required dependencies for the tests though. I think setup.py has fields for this scenario.

Reply:
Sure, will look into this.